libyul/YulStack.h

```diff
@@ -16,7 +16,7 @@
 */
 // SPDX-License-Identifier: GPL-3.0
 /**
- * Full assembly stack that can support EVM-assembly and Yul as input and EVM.
+ * Full assembly stack that can support EVM-assembly and Yul as input and EVM as output.
```

Comment:
Does it actually support EVM assembly as input?

Also, I suspect that the final `and EVM` is a leftover from the time when Ewasm was also mentioned. It probably was listed here back then.

```suggestion
 * Full assembly stack that can support Yul as input.
```

Reply:
You should check what is written in `YulStack.cpp` 😅 

Yeah exactly! But I think its still ok to just write here `... and EVM as output.`

Reply:
But yeah, I can also change that.

Reply:
Oh, didn't even realize that the .cpp file had a docstring. This is redundant. I think we only really need them in the header.

I removed it.